### PR TITLE
[fix](auth)fix missing authentication

### PR DESCRIPTION
**Pull request by zddr**
## Proposed changes

Issue Number: close #xxx

- Modified permission prompt information: added display of which permission items are required to perform the current operation, removed prompt information for "current user" and "IP"
- `AdminCopyTablet` use `PrivPredicate.ADMIN` check auth, before is `PrivPredicate.OPERATOR`
- `row policy` use `PrivPredicate.GRANT`
- `storage policy` keep `PrivPredicate.ADMIN` unchanged
- `create/alter view` need has `select_priv` on base table
- `cancel alter system` need `PrivPredicate.OPERATOR`
- `drop sync materialized view` need `alter_priv` before is `drop_priv`
- `SetLdapPassVar` need `admin_priv` before only `root/admin` can do
- `ShowCatalogRecycleBin` use `PrivPredicate.ADMIN`
-  `ShowCreateRepository` use `PrivPredicate.ADMIN`
- `ShowDataRepository` use `PrivPredicate.ADMIN`
- `ShowEncryptKey` use `PrivPredicate.ADMIN`
- `ShowPlugins` use `PrivPredicate.ADMIN`
- `ShowRepositories` use `PrivPredicate.ADMIN`
- `ShowSnapshot` use `PrivPredicate.ADMIN`
- `ShowTabletsBelong` use `PrivPredicate.ADMIN`
- `ShowTranscation` use `PrivPredicate.ADMIN`
- `show create db` use `show` instead of `PrivPredicate.ALTER_CREATE_DROP`
- `cancel export`,if table size >1,use `db.PrivPredicate.SELECT`,else use `table.PrivPredicate.SELECT`
- `show stream load`,`table.PrivPredicate.LOAD`
- `show load`,if table size >1,use `db.PrivPredicate.LOAD`,else use `table.PrivPredicate.LOAD`
## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zddr** [comment]
run buildall

**Comment by zddr** [comment]
run buildall

**Comment by zddr** [comment]
run buildall

**Comment by zddr** [comment]
run buildall

**Comment by zddr** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/common/ErrorCode.java [hunk 1]
```diff
@@ -73,7 +73,13 @@ public enum ErrorCode {
     ERR_USER_LIMIT_REACHED(1226, new byte[]{'4', '2', '0', '0', '0'}, "User '%s' has exceeded the '%s' resource "
             + "(current value: %d)"),
     ERR_SPECIFIC_ACCESS_DENIED_ERROR(1227, new byte[]{'4', '2', '0', '0', '0'}, "Access denied; you need (at least "
-            + "one of) the %s privilege(s) for this operation"),
+            + "one of) the (%s) privilege(s) for this operation"),
+    ERR_DB_ACCESS_DENIED_ERROR(1227, new byte[]{'4', '2', '0', '0', '0'}, "Access denied; you need (at least "
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/common/ErrorCode.java:77]: Better using a new error code(not 1227) for new error msg

**Review by morningman — DISMISSED** [review]
LGTM

**Comment by zddr** [comment]
run buildall

**Review by BePPPower — APPROVED** [review]

**Review by morningman — APPROVED** [review]

**Pull request merged**